lib/apis/v3/ippool.go

```diff
@@ -53,6 +53,9 @@ type IPPoolSpec struct {
 	// The block size to use for IP address assignments from this pool. Defaults to 26 for IPv4 and 112 for IPv6.
 	BlockSize int `json:"blockSize,omitempty"`
 
+	// Allows IPPool to allocate for a specific node by label selector.
```

Comment:
We should update the clientv3 e2e tests to ensure setting / getting this field is done correctly.